Onboarding/TechicalOverview.md

```diff
@@ -0,0 +1,30 @@
+# CanDIG Technical Onboarding
```

Comment:
A minor file level comment here: Try to give some newline around headers. It makes the doc easier to read.

Reply:
Sure, before, after or both?

Reply:
Both I guess